views/dataset.hbs

```diff
@@ -36,6 +36,7 @@
                             "id": "string",
                             "sourceLanguage": "string",
                             "targetLanguage": "string",
+                            "possibleEvaluatorIds": ["string"],
                             "segments": ["SegmentObject"],
                             "evaluatorIds": ["string"]
```

Comment:
This doesn't need to be specified as the person uploading the dataset only needs to give the possible evaluator IDs. The evaluatorIds will be populated when evaluators complete the evaluation